release-notes.md

```diff
@@ -1,5 +1,31 @@
 # Release notes
 
+## May 20, 2019
+
+The following updates were made in this release:
+
+#### iOS
+
+iOS Core 2.2.2 is now available!
+
+* Fixed the issue where SSL connection was not kept alive.
+* Added support for ISO 8601 timestamp for Rules Engine token replacement, you can use the placeholder `~timestampz` 
+* Fixed the bug where `ACPCore.getSdkIdentities` didn't return correct value for custom analytics id.
+* Added support for migarting Privacy status from v4 to v5, if it was manually set with v4 SDK.
+* Identity: Fixed a bug where the callback was not called on getIdentifiers if there were no custom identifiers synced before
+* Identity: Fixed a bug where VID was not present in the appendToUrl callback value
+* Identity: Fixed issue where device locale was not properly read when creating network requests from Identity extension.
+
+#### Android
+
+Android Core 1.3.1 is now available!
+
+* Added support for ISO 8601 timestamp for Rules Engine token replacement, you can use the placeholder `~timestampz`
+* Fixed the bug where `MobileCore.getSdkIdentities` didn't return correct value for custom analytics id.
+* Added support for migarting Privacy status from v4 to v5, if it was manually set with v4 SDK.
+* Added support for loading cached configuraiton and cached rules on subsequent launch.
+* Identity: Fixed issue where the Analytics visitor identifier (VID) was not included in the appendVisitorInfoForURL callback value.
```

Comment:
Same comment here, if we can have a separate section:

\#### Android Identity 1.0.5
Fixed issue where the Analytics visitor identifier (VID) was not included in the `Identity.appendVisitorInfoForURL` callback value.